docs/designs/topology.md

```diff
@@ -563,11 +645,21 @@ The operator translates user's level names to keys and builds required/preferred
 **Preferred Constraints (Auto-Generated):**
 
 - Operator ALWAYS generates preferred constraint at all three levels
-- Uses key of strictest level (e.g., `"kubernetes.io/hostname"` for "host" level)
+- Uses key of strictest (narrowest) level configured in ClusterTopology
```

Comment:
Can we include a concrete example for prefered constraints. I think I get it but I think it would be helpful as a future reference for others.